libinstpatch-1.0.pc.in

```diff
@@ -1,7 +1,6 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
```

Comment:
Did you intentionally remove the `exec_prefix`? Or should we better keep `exec_prefix=${prefix}` hardcoded in the .pc file?